proto/cosmos/gov/v1/gov.proto

```diff
@@ -12,18 +12,45 @@ import "amino/amino.proto";
 
 option go_package = "cosmossdk.io/x/gov/types/v1";
 
+// ProposalType enumerates the valid proposal types.
+// All proposal types are v1.Proposal which have different voting periods or tallying logic.
+enum ProposalType {
+  // PROPOSAL_TYPE_UNSPECIFIED defines no proposal type, which fallback to PROPOSAL_TYPE_STANDARD.
+  PROPOSAL_TYPE_UNSPECIFIED = 0;
+  // PROPOSAL_TYPE_STANDARD defines the type for a standard proposal.
+  PROPOSAL_TYPE_STANDARD = 1;
+  // PROPOSAL_TYPE_MULTIPLE_CHOICE defines the type for a multiple choice proposal.
+  PROPOSAL_TYPE_MULTIPLE_CHOICE = 2;
+  // PROPOSAL_TYPE_OPTIMISTIC defines the type for an optimistic proposal.
+  PROPOSAL_TYPE_OPTIMISTIC = 3;
+  // PROPOSAL_TYPE_EXPEDITED defines the type for an expedited proposal.
+  PROPOSAL_TYPE_EXPEDITED = 4;
+}
+
 // VoteOption enumerates the valid vote options for a given governance proposal.
 enum VoteOption {
+  option allow_alias = true;
+
   // VOTE_OPTION_UNSPECIFIED defines a no-op vote option.
   VOTE_OPTION_UNSPECIFIED = 0;
-  // VOTE_OPTION_YES defines a yes vote option.
+  // VOTE_OPTION_ONE defines the first proposal vote option.
+  VOTE_OPTION_ONE = 1;
+  // VOTE_OPTION_YES defines the yes proposal vote option.
   VOTE_OPTION_YES = 1;
-  // VOTE_OPTION_ABSTAIN defines an abstain vote option.
+  // VOTE_OPTION_TWO defines the second proposal vote option.
+  VOTE_OPTION_TWO = 2;
+  // VOTE_OPTION_ABSTAIN defines the abstain proposal vote option.
   VOTE_OPTION_ABSTAIN = 2;
-  // VOTE_OPTION_NO defines a no vote option.
+  // VOTE_OPTION_THREE defines the third proposal vote option.
+  VOTE_OPTION_THREE = 3;
+  // VOTE_OPTION_NO defines the no proposal vote option.
   VOTE_OPTION_NO = 3;
-  // VOTE_OPTION_NO_WITH_VETO defines a no with veto vote option.
+  // VOTE_OPTION_FOUR defines the fourth proposal vote option.
+  VOTE_OPTION_FOUR = 4;
+  // VOTE_OPTION_NO_WITH_VETO defines the no with veto proposal vote option.
   VOTE_OPTION_NO_WITH_VETO = 4;
```

Comment:
Doesn't make sense to me that we should be sharing the values unless there is some sort of one of.

Reply:
Please see the ADR to understand the rationale